documentation/research/custom dependencies/readme.md

```diff
@@ -0,0 +1,3 @@
+This is a short list of what custom dependencies are installed for internet.nl. This helps with creating tooling
```

Comment:
This does not add up for me. Where in which process does this help? pip-sync does not use these files, nor does the new patching documentation, and these seem like a full list of all dependencies, including simple python/pip ones, instead of a short list with custom ones?

Reply:
replaced the comment with the actual reason